packages/dialect-react-ui/components/NotificationsButton/index.tsx

```diff
@@ -111,6 +111,7 @@ function WrappedNotificationsButton(props: PropTypes): JSX.Element {
 
 export default function NotificationsButton({
   channels = ['web3', 'telegram', 'sms', 'email'],
+  notifications,
```

Comment:
I think we can omit this prop destruction, since we pass the rest of the props with `{...props}`, so the WrappedNotificationsButton would look like it looked before:

```tsx
<WrappedNotificationsButton channels={channels} {...props} />
```